src/main/java/com/royalrangers/bean/achievement/UserRewardBean.java

```diff
@@ -0,0 +1,17 @@
+package com.royalrangers.bean.achievement;
+
+import com.royalrangers.model.achievement.Reward;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class UserRewardBean {
+    private Long id;
+    private Date createDate;
+    private Date updateDate;
+    private UserAchievementBean user;
+    private Reward reward;
```

Comment:
In such case please rename it to RewardBean or something similar to avoid misunderstand situations.